src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -542,6 +542,79 @@ def create_certificate(client, vault_base_url, certificate_name, certificate_pol
 create_certificate.__doc__ = KeyVaultClient.create_certificate.__doc__
 
 
+def _asn1_to_iso8601(asn1_date):
+    import dateutil
+    return dateutil.parser.parse(asn1_date.decode('utf-8'))
+
+
+def import_certificate(client, vault_base_url, certificate_name, certificate_data,
```

Comment:
It seems to me that this logic should be moved to `azure.keyvault.key_vault_client.py` (the convenience wrapper) so that it benefits all Python developers and not just CLI users.